lib/bel/translator/plugins/rdf/translator.rb

```diff
@@ -94,6 +94,16 @@ def write_rdf_to_io(format, objects, io, options = {})
         :stream => true
       )
 
+      if (@format == :turtle)
+        require 'yaml'
+
+        prefixes = YAML::load_file('config/namespace_prefixes.yml')
```

Comment:
The `config/namespace_prefixes.yml` is a relative path. This code will expect to find it relative to the working directory when you run the `bel` command.

For example if I install the gem with your PR changes and translate bel to turtle in a directory where `config/namespace_prefixes.yml` does not exist, I will receive this error:

```bash
[tony@starship openbel]$ pwd
/home/tony/projects/openbel
[tony@starship openbel]$ bel translate -i bel.rb/small_corpus.bel bel turtle
[DEPRECATION] the SKOS vocabulary will be moved to the rdf-vocab gem
for the RDF.rb 2.0 release. Use as RDF::Vocab::SKOS, or include RDF::Vocab in the RDF module.
Called from /home/tony/.gem/ruby/2.3.0/gems/bel-0.6.0/lib/bel/translator/plugins/rdf/bel_schema.rb:197

/usr/lib/ruby/2.3.0/psych.rb:471:in `initialize': No such file or directory @ rb_sysopen - config/namespace_prefixes.yml (Errno::ENOENT)
	from /usr/lib/ruby/2.3.0/psych.rb:471:in `open'
	from /usr/lib/ruby/2.3.0/psych.rb:471:in `load_file'
	from /home/tony/.gem/ruby/2.3.0/gems/bel-0.6.0/lib/bel/translator/plugins/rdf/translator.rb:100:in `write_rdf_to_io'
	from /home/tony/.gem/ruby/2.3.0/gems/bel-0.6.0/lib/bel/translator/plugins/rdf/translator.rb:31:in `write'
	from /home/tony/.gem/ruby/2.3.0/gems/bel-0.6.0/lib/bel/translate.rb:53:in `translate'
	from /home/tony/.gem/ruby/2.3.0/gems/bel-0.6.0/bin/bel:304:in `run'
	from /home/tony/.gem/ruby/2.3.0/gems/bel-0.6.0/bin/bel:516:in `<top (required)>'
	from /home/tony/.gem/ruby/2.3.0/bin/bel:23:in `load'
	from /home/tony/.gem/ruby/2.3.0/bin/bel:23:in `<main>'
```

Instead maybe we can put `config/namespace_prefixes.yml` under the `$projectRoot/lib/bel/translator/plugins/rdf` path and load it relative to the translator.rb file.